examples/src/scala/com/twitter/heron/examples/streamlet/scala/ScalaClassicalMusicTopology.scala

```diff
@@ -0,0 +1,103 @@
+//  Copyright 2018 Twitter. All rights reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+package com.twitter.heron.examples.streamlet.scala
+
+import scala.util.Random
+
+import com.twitter.heron.examples.streamlet.scala.common.ScalaTopologyExampleUtils
+import com.twitter.heron.streamlet.scala.{Builder, Runner}
+import com.twitter.heron.streamlet.{Config, JoinType, WindowConfig}
+
+/**
+  * Classical Music Model
+  */
+case class ClassicalMusic(composer: String,
```

Comment:
use default arguments directly

Reply:
`Kryo` needs explicit defined `no-arg constructor` for Serialization so this has been added for it

Reply:
got it. kk then.